src/lib/rules/html-checker/html-checker.ts

```diff
@@ -78,7 +78,7 @@ const rule: IRuleBuilder = {
                when testing the rule and `import` doesn't work here. */
             const htmlChecker = require('html-validator');
 
-            scanOptions.data = response.body.content;
+            scanOptions.data = response.body.rawContent.toString();
```

Comment:
I don't think this change is related to edge 15. Maybe put it as a different commit?